test/page-detect.ts

```diff
@@ -391,3 +391,11 @@ test('isSingleTagPage', urlMatcherMacro, pageDetect.isSingleTagPage, [
 	'https://github.com/sindresorhus/refined-github',
 	'https://github.com/sindresorhus/refined-github/graphs'
 ]);
+
+test('isEditingFile', urlMatcherMacro, pageDetect.isEditingFile, [
+	'https://github.com/sindresorhus/refined-github/edit/master/readme.md',
+	'https://github.com/sindresorhus/refined-github/edit/ghe-injection/source/background.ts'
+], [
+	'https://github.com/sindresorhus/refined-github',
+	'https://github.com/sindresorhus/refined-github/pulls'
```

Comment:
Tests need to include some tricky urls like a user called `edit`, a repo called `edit`, a file called `edit` and so on

Reply:
Done! 😊